src/protocol/http/org/apache/jmeter/protocol/http/curl/BasicCurlParser.java

```diff
@@ -176,20 +138,18 @@
         private List<String> optionsNoSupport = new ArrayList<>();
         private List<String> optionsInProperties = new ArrayList<>();
         private Map<String, String> proxyServer = new LinkedHashMap<>();
-        private String resolverDNS;
+        private String dnsresolver;
```

Comment:
This variable is made up of two words, so use camel-case: `dnsResolver` (basically the same idea as with `caCert`).